beacon_node/src/config.rs

```diff
@@ -747,19 +747,22 @@ pub fn get_config<E: EthSpec>(
     }
 
     if cli_args.is_present("disable-proposer-reorgs") {
-        client_config.chain.re_org_threshold = None;
+        client_config.chain.re_org_head_threshold = None;
+        client_config.chain.re_org_parent_threshold = u64::MAX;
```

Comment:
with the `parent_strong` check i believe disabling proposer reorgs requires setting `re_org_parent_threshold` to `u64:MAX`

Reply:
I think we can set it to `None` can't we? If re-orgs are disabled then all the logic will bail out before it attempts to read the `re_org_parent_threshold`

Reply:
yes, as long as the `weak_head` check comes before the `parent_strong` check, it shouldn't matter. But if the order of those checks were changed, setting the  `parent_threshold` to `None` could cause some nasty side effects

Reply:
I don't think this is an issue, because the interface for `ProtoArrayForkChoice::get_proposer_head` requires two `ReOrgThreshold`s, so we're forced to handle the option. At the moment we handle the options fine by returning immediately if either is `None`. I think it would take quite a few mistakes to refactor this into something broken

Reply:
Updated in 2a83bdbafaf8561997d8f3cbc21943bfd233a177